features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -519,7 +519,7 @@
   "event.default_listener.governance_identity_store.enable_hybrid_data_store": false,
   "event.default_listener.application_authentication.priority": "11",
   "event.default_listener.application_authentication.enable": true,
-  "event.default_listener.mutual_tls_authenticator.priority": "158",
+  "event.default_listener.mutual_tls_authenticator.priority": "9999",
```

Comment:
private_key_jwt_authenticator is configured with priority "899". Lets put this above that with a number like 923. 

also lets create a migration issue to track this change in the documentation. Please link the issue to the pr as well.

Reply:
Fixed in b326cd84b23524bfa4c6d9761aded519ae036d2a and linked the created issue to the PR as well.